tests/integration/test_ethereum_tester.py

```diff
@@ -190,6 +190,57 @@ def test_eth_getTransactionReceipt_unmined(self, eth_tester, web3, unlocked_acco
             eth_tester.enable_auto_mine_transactions()
             eth_tester.mine_block()
 
+    # TODO: replace auto mining switch with decorator
+    # TODO: replicate for goethereum tests
+    def test_eth_replaceTransaction(self, eth_tester, web3, unlocked_account):
+        eth_tester.disable_auto_mine_transactions()
+        try:
+            super(TestEthereumTesterEthModule, self).test_eth_replaceTransaction(
```

Comment:
Now that we're in py3-only, we can do `super()`. Yay :)

Makes the decorator cleaner, too.